.github/workflows/formatting.yml

```diff
@@ -0,0 +1,15 @@
+name: Code formatting
+on: push
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+      - name: Install dependencies
+        run: npm ci
+      - name: Run prettier
+        run: 'npm run prettier:js'
```

Comment:
**suggestion (non-blocking):** Doesn't have to be in this PR but it might be a good idea to add a minimal prettier config even if we're just using defaults. Some editors need a config in order to run prettier, and it's just helpful to have at the project root to signal intent.

Reply:
Agreed with clear intent. 

Added `.prettierignore` and `.prettierrc` in 6342c0d.